package.json

```diff
@@ -17,16 +17,18 @@
     "test:e2e:ci": "start-server-and-test serve:dist http://localhost:8080 cy:run",
     "test": "npm run test:e2e:ci",
     "test:unit": "vue-cli-service test:unit",
-    "commit": "git-cz",
-    "lint": "vue-cli-service lint",
+    "lint": "eslint src tests build --ext .ts,.tsx,.vue,.js,.jsx",
+    "lint:fix": "npm run lint -- --fix",
     "install:local": "npm install file:../x/packages/x-components/empathyco-x-components-3.0.0-alpha.55.tgz --no-save",
     "json:csv": "json-csv ./src/i18n/messages",
     "csv:json": "csv-json ./output ./src/i18n/messages"
   },
   "dependencies": {
-    "@empathyco/x-adapter": "^7.0.0-alpha.9",
-    "@empathyco/x-components": "^3.0.0-alpha.65",
-    "@empathyco/x-types": "^10.0.0-alpha.14",
+    "@empathyco/x-adapter": "^7.0.0-alpha.23",
+    "@empathyco/x-components": "^3.0.0-alpha.84",
+    "@empathyco/x-deep-merge": "^1.3.0-alpha.11",
+    "@empathyco/x-types": "^10.0.0-alpha.20",
+    "@empathyco/x-utils": "^0.1.0-alpha.10",
```

Comment:
Can we update to the latest x-components version? Specially if we wait until [this pull request](https://github.com/empathyco/x/pull/454) gets merged, as it makes some stuff from the search `API` optional, fixing the current broken status due to the changes made in it.